Helper/Order.php

```diff
@@ -726,37 +759,20 @@ private function validateTransition($prevTransactionState, $newTransactionState)
     /**
      * Record total amount mismatch between magento and bolt order.
      * Log the error in order comments and report via bugsnag.
+     * Put the order ON HOLD if it's a mismatch.
      *
      * @param OrderModel $order
      * @param \stdClass $transaction
      * @return void
      */
-    private function checkTotalsMismatch($order, $transaction)
+    private function isTotalsMismatch($order, $transaction)
```

Comment:
given this method has side effect, I prefer calling it `checkTotalsMismatch`

Reply:
done